app/content.tsx

```
@@ -0,0 +1,124 @@
+  //       console.error('Error parsing songs:', e);
+  //     }
+  //   }
+  // }
+
+  // // shuffle songs
+  // songs = songs.sort(() => Math.random() - 0.5);
+
+  return (
+    <div className="home w-full flex flex-col gap-8">
+      <h1 className="text-lg font-bold">Chào mừng đã quay lại, {getCookie("user_name")!}</h1>
+      <h2 className="text-lg font-bold">Bài hát vừa nghe</h2>
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)